src/streamlink_cli/constants.py

```diff
@@ -13,7 +13,8 @@
 SUPPORTED_PLAYERS = {  # these are the players that streamlink knows how to set the window title for with `--title`. key names are used in help text
     # name: possible binary names (linux/mac and windows)
     "vlc": ["vlc", "vlc.exe"],
-    "mpv": ["mpv", "mpv.exe"]
+    "mpv": ["mpv", "mpv.exe"],
+    "pot": ["potplayer", "potplayermini64.exe", "potplayermini.exe"]
```

Comment:
is there even a `potplayer` binary? as this player is windows only.

Reply:
Removed

Reply:
Now tests are failing

Reply:
Reverted back to previous binary name.